django/db/backends/utils.py

```diff
@@ -114,6 +114,28 @@ def _executemany(self, sql, param_list, *ignored_wrapper_args):
             return self.cursor.executemany(sql, param_list)
 
 
+def _log_query(connection, start, stop, sql, params=None):
+    duration = stop - start
+    connection.queries_log.append(
+        {
+            "sql": "%s" % sql,
+            "time": "%.3f" % duration,
+        }
+    )
+    extra = {
+        "duration": duration,
+        "sql": sql,
+        "params": params,
+        "alias": connection.alias,
+        "format_sql": connection.ops.format_debug_sql,
```

Comment:
I'm a little worried about holding a reference to `connection.ops`. Since we haven't shipped `format_debug_sql()` yet, maybe we can reimagine it as a static method.

Suppose we need to do that anyway, since we're calling it with just `sql` as an arg.

Reply:
Great point :dart: 
I removed this adding altogether.